<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewConfigDialog</class>
 <widget class="QDialog" name="NewConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New config file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Creating a copy of {orig_config_file_name}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>New config name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="configNameLine"/>
     </item>
     <item>
      <widget class="QLabel" name="suffixLabel">
       <property name="text">
        <string>.{config_suffix}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
